### PR TITLE
Content problems - files problems fixed

### PR DESCRIPTION
**Pull request by raskin-aleksandr**

### CODE REVIEW

File: org/openscore/slang/base/network/ping.sl [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (meirwah) [inline org/openscore/slang/base/network/ping.sl]: how is this related to the Pull request (Content problems - files problems fixed )?

File: org/openscore/slang/base/network/ping_hosts.sl [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (meirwah) [inline org/openscore/slang/base/network/ping_hosts.sl]: how is this related to the Pull request (Content problems - files problems fixed )?

File: org/openscore/slang/base/properties/mail.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (meirwah) [inline org/openscore/slang/base/properties/mail.yaml]: how is this related to the Pull request (Content problems - files problems fixed )?

**Comment by orius123** [comment]
I think that in case that result is on a boolean var iable you can emmit the `== True` i.e instead of
`SUCCESS: res ==True` you can write
`SUCCESS : res`

 Can you please pull from master so I will be able to merge from github, please.
